fdk/headers.py

```diff
@@ -31,11 +32,14 @@ def encap_headers(headers, status=None):
     if headers is not None:
         for k, v in headers.items():
             if (k == constants.CONTENT_TYPE or
+                    k == constants.FN_FDK_VERSION or
                     k.startswith(constants.FN_HTTP_PREFIX)):
                 new_headers[k] = v
             else:
                 new_headers[constants.FN_HTTP_PREFIX + k] = v
 
     if status is not None:
         new_headers[constants.FN_HTTP_STATUS] = str(status)
+
+    new_headers[constants.FN_FDK_VERSION] = version.VERSION
```

Comment:
can we do this solely in fdk/response.py so as to not have it in two places (this one would only be for gateway)?

Reply:
done